src/constants.py

```diff
@@ -2,3 +2,9 @@ class SettingFileNames:
     GENERAL_FILE_NAME = "general.json"
     MONITORING_GROUPS_FILE_NAME = "monitoring_groups.json"
     RECIPIENTS_FILE_NAME = "recipients.json"
+
+
+class NotificationServiceConfig:
```

Comment:
I'm not sure if we need this specific values here.
If it's specific to lambda_test_notification_service - let's place those inside lambda code.
If not - let's discuss:
alert_header (or generally speaking "message subject", right?) - should be passed from a lambda which calls notification lambda (either alerting lambda or digest lambda).

Reply:
Agree. At the beginning of development, I thought it would be located in the constants section, but then we discussed that it is not necessary and all these variables will be passed to this lambda.